controller/src/main/java/cern/jarrace/controller/Application.java

```diff
@@ -0,0 +1,25 @@
+package cern.jarrace.controller;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+
+/**
+ * Main entry point for the controller, instanciates an embedded Jetty container server, loads the spring
+ * {@link org.springframework.web.servlet.DispatcherServlet} and configures all the
+ * {@link org.springframework.web.bind.annotation.RestController}s
+ * @author timartin
+ */
+
+@SpringBootApplication
+public class Application {
+    public static void main(String args[]) {
```

Comment:
Wouldnt it be nicer to have the main in a separate class than the spring boot configuration?